### DISCUSSION TITLE
How to enable ssl-feature

### DISCUSSION THREAD

**Discussion by hasanramp** [RustPython/RustPython#3496] [Q&A] [answered]
I just installed rustpython using cargo. In the readme I found that to install pip you need tor run ```rustpython --install-pip``` but it says that for this you need to enable ```ssl feature```. How do I enable this ```ssl feature```? I am using Ubuntu.

**Comment by coolreader18** [comment, marked as answer]
Just run `cargo build --release` with `--features ssl`
